crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -149,6 +152,20 @@ def cancel_allowed?
       ].include?(current_step) && !running?(:admin)
     end
 
+    def save_crowbar_backup(backup_location = "")
```

Comment:
Well, `reasonable default` here is strange. Library itself should not know about that path if if it wasn't provided with the call, there's no reason to guess...